Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+
+def simulate_round(pairings):
+    # Simulate games for each pairing
+    results = {}
+    for pair in pairings:
+        # Simulate game between pair[0] and pair[1]
+        # Update results based on game outcome (win, loss, draw)
+        # For simplicity, let's assume random results
+        Coin_Flip = random.randint(1,2)
+        if Coin_Flip == 1:
+            Hero1 = Player_List.get(pair[0])
+            Hero2 = Player_List.get(pair[1])
+            Weight = Matchups[Hero1]["First"][Hero2]
+        elif Coin_Flip == 2:
+            Hero1 = Player_List.get(pair[0])
+            Hero2 = Player_List.get(pair[1])
+            Weight = Matchups[Hero1]["Second"][Hero2]
+        Deceider = round(random.uniform(0,1),2)
+        results[pair[0]] = 'win' if Deceider <= Weight else 'loss' if Deceider > Weight else 'draw'
+        results[pair[1]] = 'win' if results[pair[0]] == 'loss' else 'loss' if results[pair[0]] == 'win' else 'draw'
+    return results
+
+
+previous_pairings = set()
+results = {key: 0 for key in Player_List.keys()}
+Tie_Breakers = {key: 0 for key in Player_List.keys()}
+for round_num in range(int(Rounds)):
+    round_num+1
+    """print("\n" "Round", round_num+1)"""
+    pairings, previous_pairings = generate_pairings(players, previous_pairings, results)
+    """print("\n" "Pairings:", pairings)"""
+    round_results = simulate_round(pairings)
+    """print("\n" "Results:", round_results)"""
+    for player in round_results:
+        if round_results.get(player) == 'win':
+            results[player] += 1
+            Tie_Breakers[player] += (int(Rounds)-(int(round_num)-1))
+        elif round_results.get(player) == 'lose':
+            results[player] += 0
+            
+
+#Add Tiebreaker values to records to find tie breakers
+for player in results:
+    results[player] += Tie_Breakers[player]
+
+#dictionary for top 64 hero counts
+Hero_Results = []
+
+for player in results:
+    Hero_Results.append(((Player_List[player]),(results[player])))
+
+sorted_overall_results = sorted(Hero_Results, key=lambda x: x[1], reverse=True)
+
+#Dictionary for just hero names for placing
+clean_results = []
+a = 0
+
+#Strip anything but hero name from results
+while a < len(sorted_overall_results):
+    clean = str(sorted_overall_results[a])
+    clean = re.sub("[(),1234567890' ]", '', clean)
+    clean_results.append(clean)
+    a += 1
+print("\n")
+
+#Set Excel defaults to output file
+path = "Calculator Export.xlsx"
+wb_obj = openpyxl.load_workbook(path)
+wb_obj.create_sheet(Sheet_Name)
+wb_obj.active = wb_obj[Sheet_Name]
+sheet_obj = wb_obj.active
+
+#Set columns for new sheet_obj
+sheet_obj.cell(row=(1),column=(1)).value = "Placement"
+sheet_obj.cell(row=(1),column=(2)).value = "Hero"
+sheet_obj.cell(row=(1),column=(5)).value = "Top 64 Breakdown"
+sheet_obj.cell(row=(1),column=(6)).value = "Hero"
+sheet_obj.cell(row=(1),column=(7)).value = "Number"
+
+#print data to workbook
+place = 0
+top_sixtyfour = dict()

```

Comment:
The tournament size should also be an argument to the script, that will give you good practice in making the code more flexible and is generally what I would expect a script like this to be able to do. You can constrain the range of values that is permitted, that's pretty normal